lib/martilla/notifiers/slack.rb

```diff
@@ -1,4 +1,51 @@
+require 'slack-notifier'
+
 module Martilla
   class Slack < Notifier
+    def success(data)
+      slack_notify(success_txt(data))
+    end
+
+    def error(msg, data)
+      slack_notify(error_txt(msg, data))
+    end
+
+    private
+
+    def slack_notify(message)
+      begin
+        notifier.ping message
+      rescue StandardError => e
+        puts "Slack notification not sent. Error message: #{e}"
+      end
+    end
+
+    def notifier
+      ::Slack::Notifier.new(slack_webhook_url, channel: slack_channel, username: slack_username)
+    end
+
+    def slack_webhook_url
+      raise config_error('slack_webhook_url') if @options['slack_webhook_url'].nil?
+      @options['slack_webhook_url']
+    end
+
+    def slack_channel
+      @options['slack_channel'] or "#general"
+    end
+
+    def slack_username
+      @options['slack_username'] or "Martilla"
```

Comment:
```suggestion
      @options['slack_username'] || "Martilla"
```

Since I've been using the `||` operator all around the project I would rather keep it consistent instead of using the `or` operator